x-pack/test/functional/page_objects/api_keys_page.ts

```diff
@@ -126,6 +130,10 @@ export function ApiKeysPageProvider({ getService }: FtrProviderContext) {
       return await testSubjects.find('apiKeysRoleDescriptorsSwitch');
     },
 
+    async getFlyout() {
+      return await find.byClassName('euiFlyout');
+    },
+
```

Comment:
__question:__ do we use this `getFlyout` method anywhere right now?

Reply:
Great catch - Removed in latest commit!